types/mixpanel-browser/index.d.ts

```diff
@@ -70,13 +70,13 @@ export interface Config {
     test: boolean;
     verbose: boolean;
     img: boolean;
+    /**
+     * @default false
+     * @see https://github.com/mixpanel/mixpanel-js/blob/master/doc/readme.io/javascript-full-api-reference.md#mixpanelset_config
+     */
     debug: boolean;
```

Comment:
Ideally, it'd be nice if all the `Config` values showed their `default` values on hover, as well as a documentation link to find out more, but I don't have the bandwidth for that at the moment.